### PR TITLE
Fixes for restclient and deep clone, plus reverse order of feed items (oldest came first)

### PR DESCRIPTION
**Pull request by mironov**

### CODE REVIEW

**Comment by jookyboi** [comment]
Awesome. Thanks for being the first pull request to address these issues.

File: lib/delayed_jobs/process_new_articles_job.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def call_webhook(article_hash, rss_feed)
     output_settings = nil
 
     if rss_feed['output']
-      output_settings = rss_feed['output'].clone # don't change the original
+      Marshal.load(Marshal.dump(rss_feed['output'])) # don't change the original
```
Comment (jookyboi) [inline lib/delayed_jobs/process_new_articles_job.rb]: Shouldn't the Marshal.load be loading into output_settings?

Comment (mironov) [inline lib/delayed_jobs/process_new_articles_job.rb]: Wow, overlooked while copying from one file to another.
Fixed.

**Comment by jookyboi** [comment]
This is ready to go.

**Pull request merged**